### PR TITLE
Swift 5 Implementation

### PR DESCRIPTION
**Pull request by chessinb**
https://github.com/nessieisreal/nessie-ios-sdk-swift/issues/12

### CODE REVIEW

**Comment by chessinb** [comment]
@borikanes Here is the PR!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Nessie-iOS-Wrapper
-### Note: This SDK is Swift 3.0 thanks to [UberJason](https://github.com/UberJason). Please use Xcode 8 and later 😀
+### Note: This SDK is Swift 5.0, please use Xcode 8 and later 😀
```
Comment (borikanes) [inline README.md]: Something small but can you say Xcode 10 and later here?

**Review by borikanes — APPROVED** [review]
LGTM. Made a comment about the README specifying an old xcode version.

**Review by chrislonge — COMMENTED** [review]
Why not bump to 5.1?

**Comment by chessinb** [comment]
@chrislonge This was to not force people to have Xcode 11 installed, in case some are using Xcode 10.2 still it would still work for them.

File: README.md (hunk 1 above)
Comment (chessinb) [reply README.md]: Fixed, thanks!

**Pull request closed without merging**